assets/js/components/notifications/UserInputSettings.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * UserInputSettings component.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId as useInstanceID } from '@wordpress/compose';
```

Comment:
For what it's worth, you _should_ be able to import this directly as `useInstanceId`—the linter should be smart enough to pick up that non-default imports don't need to match our capitalisation rules… and in this case I think it'd better they don't, because it makes grepping for the function according to how it's exported (essentially `@wordpress/compose`'s exposed API) easier.

Reply:
Update: I see this DOES breaking naming rules. But that kinda sucks; it's not great having to do this kind of renaming dance.

I'll include that in my tweaks for #2154 or something. This shouldn't be necessary, but I checked locally and it is. 😢

Reply:
@tofumatt that would be great if we could address this. I already had to add some property renaming for RTL (which isn't great but better than using ignores everywhere):
https://github.com/google/site-kit-wp/blob/e1bd73a71e81ce09aab24e29691da6d8ddb2ab07/tests/js/test-utils.js#L52-L60

Reply:
Filed #2346